packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1162,12 +1162,18 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
     assert(reason != null);
     if (_snackBars.isEmpty || _snackBarController.status == AnimationStatus.dismissed)
       return;
+    final MediaQueryData mediaQuery = MediaQuery.of(context);
     final Completer<SnackBarClosedReason> completer = _snackBars.first._completer;
-    _snackBarController.reverse().then<void>((Null _) {
-      assert(mounted);
-      if (!completer.isCompleted)
-        completer.complete(reason);
-    });
+    if (mediaQuery.accessibleNavigation) {
```

Comment:
Missing unit test for this scenario (hideCurrentSnackbar is called with/without accessibleNavigation).

Reply:
Should be L522-523 of snack_bar_test.dart to cover the true case, other case is handled previously

Reply:
right missed that.